src/main/pages/che-7/end-user-guide/assembly_converting-a-che-6-workspace-to-a-che-7-devfile.adoc

```diff
@@ -19,16 +19,202 @@ summary:
+      "mixins": []
+    }
+  ],
+  "name": "wksp-jhwp",
+  "commands": [
+    {
+      "commandLine": "scl enable rh-maven33 'mvn compile vertx:debug -f ${current.project.path} -Dvertx.disableDnsResolver=true'",
+      "name": "debug",
+      "attributes": {
+        "goal": "Debug",
+        "previewUrl": "${server.8080/tcp}"
+      },
+      "type": "custom"
+    },
+    {
+      "commandLine": "scl enable rh-maven33 'mvn compile vertx:run -f ${current.project.path} -Dvertx.disableDnsResolver=true'",
+      "name": "run",
+      "attributes": {
+        "goal": "Run",
+        "previewUrl": "${server.8080/tcp}"
+      },
+      "type": "custom"
+    },
+    {
+      "commandLine": "scl enable rh-maven33 'mvn clean install -f ${current.project.path}'",
+      "name": "build",
+      "attributes": {
+        "goal": "Build",
+        "previewUrl": ""
+      },
+      "type": "mvn"
+    },
+    {
+      "commandLine": "mvn -Duser.home=${HOME} -f ${CHE_PROJECTS_ROOT}/vertx-http-booster clean package",
+      "name": "vertx-http-booster:build",
+      "attributes": {
+        "goal": "Build",
+        "previewUrl": ""
+      },
+      "type": "mvn"
+    },
+    {
+      "commandLine": "mvn -Duser.home=${HOME} -f ${CHE_PROJECTS_ROOT}/vertx-http-booster vertx:run",
+      "name": "vertx-http-booster:run",
+      "attributes": {
+        "goal": "Run",
+        "previewUrl": "${server.8080/tcp}"
+      },
+      "type": "mvn"
+    }
+  ],
+  "links": []
+}
+----
++++ </div></details> +++
+
++++ <details><summary> +++
+Che 7 devfile
++++ </summary><div> +++
+----
+metadata:
+  name: testing-workspace
+projects:
+  - name: java-web-vertx
+    source:
+      location: 'https://github.com/che-samples/web-java-vertx'
+      type: git
+components:
+  - id: redhat/java/latest
+    type: chePlugin
+  - mountSources: true
+    endpoints:
+      - name: 8080/tcp
+        port: 8080
+    memoryLimit: 512Mi
+    type: dockerimage
+    volumes:
+      - name: m2
+        containerPath: /home/user/.m2
+    alias: maven
+    image: 'quay.io/eclipse/che-java8-maven:nightly'
+    env:
+      - value: >-
+          -XX:MaxRAMPercentage=50.0 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10
+          -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4
+          -XX:AdaptiveSizePolicyWeight=90 -Dsun.zip.disableMemoryMapping=true
+          -Xms20m -Djava.security.egd=file:/dev/./urandom -Duser.home=/home/user
+        name: JAVA_OPTS
+      - value: $(JAVA_OPTS)
+        name: MAVEN_OPTS
+apiVersion: 1.0.0
+commands:
+  - name: maven build
+    actions:
+      - workdir: '${CHE_PROJECTS_ROOT}/java-web-vertx'
+        type: exec
+        command: 'mvn -Duser.home=${HOME} clean install'
+        component: maven
+  - name: run app
+    actions:
+      - workdir: '${CHE_PROJECTS_ROOT}/java-web-vertx'
+        type: exec
+        command: >
+          JDBC_URL=jdbc:h2:/tmp/db \
+
+          java -jar -Xdebug
+          -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=5005 \
+
```

Comment:
Blank line? And I may be wrong but there is no corresponding command in the Che 6 example

Reply:
This is how the devfile is generated by default, I didn't change anything. So should I remove whole this command `run app`?